napari/utils/misc.py

```diff
@@ -275,7 +277,7 @@ def __call__(
         *,
         module=None,
         qualname=None,
-        type=None,
+        type=None,  # noqa: A002
```

Comment:
Why is this unchanged while other occurrences of `type` are?

Reply:
Because I failed to examine the code base where it is used